Makefile

```diff
@@ -0,0 +1,29 @@
+test:
+	go test -v ./...
+
+build: test
+	go build
+
+check: lint vet fmtcheck ineffassign
+
+lint:
+	golint -set_exit_status .
+
+vet:
+	go vet
```

Comment:
I recommend developing at tip until Go 1.10 is out, for a number of plugin bug fixes that are in the tree. A side effect of that is that go vet is now run every time go test is run, so there's no need for extra steps any more.